### PR TITLE
Added adafruit pip packages and scipy for Ubuntu Bionic

### PR DESCRIPTION
**Pull request by nicolaje**
Replaces issue #18328 where it now contains only adafruit related python packages.

The INA219 is an i2c current sensor, that can be used to monitor current in robots:
https://pypi.org/project/pi-ina219/

The ADS1015 is an i2c analog to digital converter, Adafruit sells a board with this sensor along with the python driver:
https://pypi.org/project/Adafruit-ADS1x15/

Sympy is a package for symbolic mathematics with Python:
https://pypi.org/project/sympy/
python-sympy Ubuntu: https://packages.ubuntu.com/bionic/python-sympy

### CODE REVIEW

File: rosdep/python.yaml
Comment (mikaelarguedas) [inline rosdep/python.yaml]: Can you please also add artful and cosmic ?

**Review by mikaelarguedas — COMMENTED** [review]
can you please provide a brief description of + use case for the newly added packages as per [contribution guildelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions)?

Please also provide a link to the packages on packages.ubuntu.com or pypi.org.

Thanks!

File: rosdep/python.yaml
Comment (nicolaje) [reply rosdep/python.yaml]: Done !

**Comment by nicolaje** [comment]
@mikaelarguedas I updated the PR description with the link & use cases of the newly added packages

**Comment by mikaelarguedas** [comment]
python-sympy Ubuntu: https://packages.ubuntu.com/bionic/python-sympy

**Review by mikaelarguedas — APPROVED** [review]

**Comment by nicolaje** [comment]
@mikaelarguedas can we merge?

**Comment by nicolaje** [comment]
@mikaelarguedas ?

**Comment by mikaelarguedas** [comment]
@nicolaje New rosdep keys need 2 approvals to be merged, this is why we needed another rosdistro maintainer to review and merge this

**Pull request merged**
